stable/kommander-ui/templates/hooks-roles.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "kommander.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "kommander.labels" . | indent 4 }}
+  annotations:
+    helm.sh/hook: "pre-install"
```

Comment:
Why do these need to be pre install hooks?

Reply:
I did not write this. It seems like it was some lifecycle reason?